react/features/base/ui/components/web/Dialog.tsx

```diff
@@ -282,8 +282,14 @@ const Dialog = ({
                                 onClick = { onClose } />
                         )}
                     </div>
-                    <div className = { classes.content }>{children}</div>
-                    <div className = { classes.footer }>
+                    <div
+                        className = { classes.content }
+                        data-autofocus-inside = 'true'>
+                        {children}
```

Comment:
couldn't we use react/features/base/util/react-focus-lock-wrapper.js ? or it's not applicable here?

Reply:
Not really. Here we want the whole dialog in focus, but we want the auto focus to be on the second focusable element. The first is the close icon in the header

Reply:
Also, the file `react/features/base/util/react-focus-lock-wrapper.js` seems to complicate things unnecessarily. After the dialog transition is done I'll remove it